spec/config.json

```diff
@@ -452,6 +513,22 @@
         }
       }
     },
+    "client": {
+      "default": {
+        "tls": {
+          "$ref": "#/definitions/client_tls_config"
+        }
+      },
+      "back_channel_logout": {
+        "tls": {
+          "allOf": [
```

Comment:
I don't think you need the `allOf` here, just using `ref` like above should be enough.